pkg/common/args.go

```diff
@@ -68,10 +66,7 @@ func ValidateArgs() {
 			if !strings.HasPrefix(key, "--") && strings.Count(key, "--") != 1 {
 				log.Fatalf("Expected key [%s] in [%s] to start with prefix --", key, extraArgs)
 			}
-			updatedExtraArgs = fmt.Sprintf("%s%s%s", updatedExtraArgs, extraArgsSeperator, kv)
```

Comment:
if validation passes, we do not need to set "extra-args" to string type. this was not the right way since we were storing []string{} and then string string type in same flag.